gnosis/safe/addresses.py

```diff
@@ -893,4 +901,10 @@
     EthereumNetwork.OASIS_SAPPHIRE: [
         ("0xC22834581EbC8527d974F8a1c97E1bEA4EF910BC", 325632),  # v1.3.0
     ],
+    EthereumNetwork.NEON_EVM_DEVNET: [
+        ("0xC22834581EbC8527d974F8a1c97E1bEA4EF910BC", 205146874),  # v1.3.0
+    ],
+    EthereumNetwork.NEON_EVM_MAINNET: [
+        ("0xC22834581EbC8527d974F8a1c97E1bEA4EF910BC", 203993869),  # v1.3.0
```

Comment:
This block is not having transactions, can you double check?
https://neonscan.org/block/203993869#transactions

Reply:
Well, it has 1 tx - https://neonscan.org/tx/0x13823780528d1281bd802a8744bae19f14b62f941f7fdf936b1b67a4c85b92a1
Probably again was an explorer temporary glitch